__tests__/servertesting.test.js

```diff
@@ -36,3 +36,17 @@ describe("/get/api/topics", () => {
       });
   });
 });
+describe("/get/api ", () => {
+  it("200: Should return a list of all endpoints", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((data) => {
+        data = JSON.parse(data.text);
+        console.log(data);
+        expect(data).toHaveProperty("GET /api");
+        expect(data).toHaveProperty("GET /api/topics");
+        expect(data).toHaveProperty("GET /api/articles");
+      });
```

Comment:
This test will fail when you make your next endpoint. Consider how you can use the whole json file and compare it to your response to make this test more robust. Changing this test for each endpoint goes against the TDD principles we have taught on the course.